### PR TITLE
Fix loading icon bug on IE

### PR DESCRIPTION
**Pull request by scott1702**
Unfortunately IE can't handle css animating SVGs, this just turns the <svg> into <span>s so we can animate them in IE.

Fixes https://github.com/silverstripe/silverstripe-framework/issues/5283

### CODE REVIEW

**Comment by chillu** [comment]
@scott1702 This hasn't been tested with other button style variations I think, the dots are grey regardless of the button text colour (cc @clarkepaul)

image

**Comment by scott1702** [comment]
Should be good now @chillu

**Comment by chillu** [comment]
Broken up in two lines for IE10 on "Edit Campaign": http://take.ms/8rzqa

**Comment by scott1702** [comment]
Talked this over with @clarkepaul around how we could handle this, came to the decision of using the same indicator with smaller circles. This change has been added and tested with buttons as small as 'icon-only'.

Third time lucky @chillu? 😉

**Comment by clarkepaul** [comment]
Looked good to me and tested in IE10.

File: admin/client/src/components/FormAction/FormAction.scss [hunk 1]
```diff
@@ -143,8 +139,26 @@
...
 
-  svg circle {
-    fill: $body-color;
+  .btn__circle {
+    background: $btn-secondary-color;
+  }
+}
+
+.btn-danger {
+  .btn__circle {
```
Comment (dhensby) [inline admin/client/src/components/FormAction/FormAction.scss]: you're not meant to nest elements in BEM

**Comment by dhensby** [comment]
needs rebasing :/

**Comment by scott1702** [comment]
Rebased :)

**Comment by dhensby** [comment]
Thanks

**Pull request merged**